### PR TITLE
Bulk import batch size

### PR DESCRIPTION
**Pull request by Blacksmoke16**
PR for #190

### CODE REVIEW

File: src/granite_orm/transactions.cr [hunk 1]
```diff
@@ -10,25 +10,31 @@ module Granite::ORM::Transactions
     # The import class method will run a batch INSERT statement for each model in the array
     # the array must contain only one model class
     # invalid model records will be skipped
-    def self.import(model_array : Array(self))
+    def self.import(model_array : Array(self), batch_size : Int32 = model_array.size)
       begin
-        @@adapter.import(table_name, primary_name, fields.dup, model_array)
+        model_array.each_slice(batch_size, true) do |slice|
+          @@adapter.import(table_name, primary_name, fields.dup, slice)
```
Comment (robacarp) [inline src/granite_orm/transactions.cr]: Can you get away with calling `#dup` on fields only once and reusing that duped copy?

Comment (Blacksmoke16) [reply src/granite_orm/transactions.cr]: I'll take a look, good catch.

File: src/granite_orm/transactions.cr [hunk 2]
```diff
@@ -12,8 +12,9 @@ module Granite::ORM::Transactions
     # invalid model records will be skipped
     def self.import(model_array : Array(self), batch_size : Int32 = model_array.size)
       begin
+        fields_duplicate = fields.dup
         model_array.each_slice(batch_size, true) do |slice|
-          @@adapter.import(table_name, primary_name, fields.dup, slice)
+          @@adapter.import(table_name, primary_name, fields_duplicate, slice)
```
Comment (drujensen) [inline src/granite_orm/transactions.cr]: 🥇

**Comment by Blacksmoke16** [comment]
Also @robacarp @drujensen do you have any better ideas to test that two separate inserts were executed?  Current test is just testing that everything was inserted correctly, which would at least mean it worked?

**Comment by drujensen** [comment]
@Blacksmoke16 maybe look at the `created_at` timestamp? I'm not sure that is reliable though.

**Pull request merged**